src/OpenLoco/OpenLoco.cpp

```diff
@@ -922,7 +923,7 @@ namespace OpenLoco
 
                     if (today.year <= 2029)
                     {
-                        call(0x0046E239);
+                        Economy::sub_46E239();
```

Comment:
`updateMonthly`

Reply:
Sounds like a good name for it. I'll address it in #362